### ISSUE TITLE
Chat control buttons are lacking accessibility descriptions

### ISSUE THREAD

**Issue by vojtapolasek** [open-webui/open-webui#6732]
# Bug Report

---

## Installation Method

I use podman and I use the latest image.

## Environment

- **Open WebUI Version:** 0.3.35
- **Ollama (if applicable):** not relevant

- **Operating System:** Fedora 40
- **Browser (if applicable):** Firefox 132

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When accessing the buttons around the chat area with a screen reader, I expect to get their description / label announced.

## Actual Behavior:

When accessing buttons, they are announced only as "button".

## Description

**Bug Summary:**
Buttons around the chat area are lacking accessible descriptions.

## Reproduction Details

**Steps to Reproduce:**
Start a chat.
Start a screen reader.
Use the TAB key to move through the interface until you reach buttons for regenerating chat, thumbs up / thumbs down etc. Move through buttons with tab.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

I am willing to test your updates if you provide it in a contianer image or something like that.

**Comment by tjbck** [comment]
#2790
